app/models/feedback.rb

```diff
@@ -14,7 +14,15 @@ class Feedback < ApplicationRecord
 
   validate :check_for_spam
   def check_for_spam
-    errors.add(:base, ts("^This comment looks like spam to our system, sorry! Please try again, or create an account to comment.")) unless check_for_spam?
+    errors.add(:base, ts("This report looks like spam to our system!")) unless check_for_spam?
+  end
+
+  def check_for_spam?
+    self.approved = logged_with_matching_email? || !Akismetor.spam?(akismet_attributes)
+  end
+
+  def logged_with_matching_email?
```

Comment:
Should this be `logged_in_with_matching_email`?

Reply:
true, renaming in the next commit